cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -1367,7 +1370,7 @@ endif()
 # ----------------------------------------------------------------------
 # mimalloc - Cross-platform high-performance allocator, from Microsoft
 
-if(ARROW_MIMALLOC)
+macro(build_mimalloc)
   message(STATUS "Building (vendored) mimalloc from source")
   # We only use a vendored mimalloc as we want to control its build options.
```

Comment:
Is this comment obsolete? Does conda-forge uses the same build options as we do?

Reply:
😞 Thought so but missed `MI_OVERRIDE`, not sure whether we can set that on conda-forge